code-build-actions/build-js-sdk/action.yaml

```
@@ -58,7 +58,7 @@ runs:
         rm -rf openapitools.json templates-js .openapi-generator-ignore .openapi-generator git_push.sh
     - name: Commit SDK changes to the PR
       uses: EndBug/add-and-commit@v9
-      if: ${{ !inputs.dry-run }}
+      if: ${{ !contains(inputs.dry-run, 'false') }}
```

question: why not this?
```suggestion
      if: ${{ inputs.dry-run != 'false' }}
```

---

Not really, nope! I just figured being explicit to avoid type coercion.